assets/js/modules/analytics-4/components/dashboard/EnhancedMeasurementActivationBanner/SetupBanner.js

```diff
@@ -100,6 +106,12 @@ export default function SetupBanner( {
 		onSubmitSuccess();
 	}, [ setValues, submitChanges, onSubmitSuccess ] );
 
+	useMount( () => {
+		if ( usingProxy && ! isDismissed ) {
+			triggerSurvey( 'view_enhanced_measurement_cta', { ttl: 0 } );
```

Comment:
As mention in the above comment example, we need to use non-empty TTL for this survey trigger to avoid sending survey trigger requests on every page reload. Let's use `DAY_IN_SECONDS` as ttl for this survey.